### ISSUE TITLE
actor.slicer function segmentation faults when using a int64 data type as input

### ISSUE THREAD

**Issue by pjsjongsung** [fury-gl/fury#851]
## Description
If fury.actor.slicer is passed a int64 type image, it throws a segmentation fault. Not a critical bug, but could be confusing to users if they encounter this.

## Way to reproduce
```
from fury.actor import slicer
import numpy as np
temp = np.ones((20, 20, 20)).astype(np.int64)
slicer(temp, np.eye(4))

```
Operating system is ubuntu 20.04. Tested with python 3.9 and 3.10

**Comment by Preetam-Das26** [comment]
I would like to work on this issue.

**Comment by IshanKamboj** [comment]
is this issue still reproducible? @pjsjongsung ... The given code works perfectly fine for me

**Comment by skoudoro** [comment]
Yes, I just reproduce it @IshanKamboj.  I am using macOS, maybe it is specific to the OS which would be strange. I tried with different python version and still the issue

**Comment by IshanKamboj** [comment]
> Yes, I just reproduce it @IshanKamboj. I am using macOS, maybe it is specific to the OS which would be strange. I tried with different python version and still the issue

Ohh it works perfectly fine on windows. I think the issue is specific to linux/macOS.

**Comment by pjsjongsung** [comment]
Could it be related to python version? I just tested on python 3.8 and 3.10 on windows and still get the fault.

**Comment by skoudoro** [comment]
We both tested on different python version / OS  and we still get segfault.

only @IshanKamboj do not get a segfault. Maybe he can share its full setup. (OS, python version, all packages versions / environment, etc...)

**Comment by IshanKamboj** [comment]
I am using windows 11 as my OS and python version : 3.10.11

Following are the package versions:
fury==0.10.0
vtk==9.3.0
numpy==1.26.2
scipy==1.11.4
Pillow==10.0.1
pygltflib==1.16.1
packaging==23.2
aiohttp==3.9.1

**Comment by robinroy03** [comment]
I reproduced the segmentation fault on both Ubuntu(WSL) and Windows 11.

Ubuntu(WSL) mentions `segmentation fault` on terminal but windows powershell terminates the code abruptly.

```py
from fury.actor import slicer
import numpy as np
temp = np.ones((20, 20, 20)).astype(np.int64)
slicer(temp, np.eye(4))
print("this won't be printed")
```

The last line won't be printed on windows if tried. The code crashes.

**Comment by IshanKamboj** [comment]
@robinroy03 is right ... The code ends abruptly and does not show segmentation fault on the terminal in windows
